go.mod

```diff
@@ -0,0 +1,61 @@
+module github.com/docker/libkv
+
+go 1.17
+
+require (
+	github.com/coreos/etcd v3.3.25+incompatible
+	github.com/hashicorp/consul/api v1.1.0
```

Comment:
Comparing this with the current version used in moby; https://github.com/hashicorp/consul/compare/v0.5.2...api/v1.1.0

Reply:
Here is another problem, `go mod tidy` trips over this library because v0.5.2 doesn't use models, but new version of it does. Without and there is no issue with this library at all until you run `go mod tidy`...

Reply:
```
github.com/docker/docker/libnetwork imports
	github.com/docker/libkv/store/consul imports
	github.com/hashicorp/consul/api loaded from github.com/hashicorp/consul@v0.5.2,
	but go 1.16 would fail to locate it:
	ambiguous import: found package github.com/hashicorp/consul/api in multiple modules:
	github.com/hashicorp/consul v0.5.2 (/Users/ilya/Code/moby/.gopath/pkg/mod/github.com/hashicorp/consul@v0.5.2/api)
	github.com/hashicorp/consul/api v1.1.0 (/Users/ilya/Code/moby/.gopath/pkg/mod/github.com/hashicorp/consul/api@v1.1.0)
```
@crazy-max and I tried a few different replace options, still the same issue...